### DISCUSSION TITLE
Dishwasher tabs/tablets

### DISCUSSION THREAD

**Discussion by mrcl-rckl** [ekutner/home-connect-hass#260] [Ideas]
Dear ekutner,
I integrated a Siemens dishwasher with your great integration. Would it be possible for you to integrate the counter (which i can set/see in the Home Connect app) which counts the remaining tablets for the dishwasher?

I hope you can implement it as an sensor for Home Assistant. 

Best regards

**Comment by ekutner** [comment]
Please read the README. The integration automatically adds any entities (sensors or others) that are made available by the API.

**Comment by rodehoed** [comment]
I presume they are only in the private api and thus only available to App users.

What about warnings? @ekutner ? My dishwasher nags (App) for Salt sometimes. 

BTW I gave a ⭐ already, but also would like to thank you. This integration should be the official one actually (I know the reasons why its not). Keep up the good work!

**Reply by kwbeck** [reply]
There is a way to set up binary sensors for rinse aid and salt.   Slightly complicated, but with help from others here, I got it working.  Have a look at my reply to an earlier question.  -->   https://github.com/ekutner/home-connect-hass/discussions/304#discussioncomment-8004914
